editor/src/components/editor/actions/actions.spec.browser2.tsx

```diff
@@ -271,4 +278,675 @@ describe('actions', () => {
       })
     })
   })
+  describe('PASTE_JSX_ELEMENTS', () => {
+    type PasteTest = {
+      name: string
+      startingCode: string
+      elements: (renderResult: EditorRenderResult) => Array<ElementPaste>
+      pasteInto: ReparentTargetParent<ElementPath>
+      want: string
+    }
+    const tests: Array<PasteTest> = [
+      {
+        name: 'a single element',
+        startingCode: `
+        <div data-uid='aaa'>
+            <div data-uid='bbb'>foo</div>
+            <div data-uid='ccc'>bar</div>
+		</div>
```

Comment:
<img width="415" alt="image" src="https://user-images.githubusercontent.com/2226774/232710716-e5c5543f-db31-4c5f-8f7e-faa0963ebc59.png">
sorry, you forgot the last lines in this file. Github does not forgive! :D

Reply:
https://github.com/concrete-utopia/utopia/pull/3544/commits/eb7f9e96b6915ef078b161c2e5c6ebcb4fa08f5e